src/Microsoft.IdentityModel.Tokens.Saml/Saml2/LogMessages.cs

```diff
@@ -12,6 +12,15 @@ namespace Microsoft.IdentityModel.Tokens.Saml2
     internal static class LogMessages
     {
 #pragma warning disable 1591
+
+        // general
+        internal const string IDX10000 = "IDX10000: The parameter '{0}' cannot be a 'null' or an empty object. "; //Q: Should we just use one set of validation errors?
```

Comment:
why do we need new `IDX????` codes?

Reply:
If this is just to use in the tests, we could simply import the `LogMessages` from the outer package, Tokens.